examples/fizz-buzz.ktn

```diff
@@ -0,0 +1,26 @@
+define buzzes(Int32 -> +IO +Fail):
+    -> n;
+    n 5 (%) 0 (=)
+    n 3 (%) 0 (=)
+
+    match
```

Comment:
Any particular reason you chose `match case true { … } case false { … }` over `if { … } else { … }`? (You can omit the `if` condition, same as `match`.)